app/controllers/idv_controller.rb

```diff
@@ -1,7 +1,30 @@
 class IdvController < ApplicationController
-  def p1
+  def register_user(args = nil)
+    agent = Proofer::Agent.new(vendor: :mock)
+    app_vars = params.slice(:first_name, :last_name, :dob)
+    applicant = Proofer::Applicant.new(app_vars)
+    session[:resolution] = agent.start(applicant)
```

Comment:
yeah, it's totally the wrong move to store it in the session. Was path of
least resistance, but it's not at all the right way to do it.

On Fri, Jun 10, 2016 at 10:48 AM, Peter Karman <notifications@github.com>
wrote:

> In app/controllers/idv_controller.rb
> <https://github.com/18F/identity-idp/pull/161#discussion_r66653796>:
>
> > @@ -1,7 +1,30 @@
> >  class IdvController < ApplicationController
> > -  def p1
> > +  def register_user(args = nil)
> > +    agent = Proofer::Agent.new(vendor: :mock)
> > +    app_vars = params.slice(:first_name, :last_name, :dob)
> > +    applicant = Proofer::Applicant.new(app_vars)
> > +    session[:resolution] = agent.start(applicant)
>
> are we ok storing the security questions in the session? what if a user
> has to leave their browser mid-session?
>
> could we store them server-side in a Redis blob instead?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/identity-idp/pull/161/files/3a2670f1e40d3f2e7828239d20829e5f297e66a5#r66653796>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABQi_87EWF1BREBVZwRQ30_JxQpagz5rks5qKaNxgaJpZM4Iyi5A>
> .
>